pages/api/randomProfile.js

```diff
@@ -0,0 +1,40 @@
+// pages/api/randomProfile.js
+require('dotenv').config();
+
+
+const { MongoClient } = require('mongodb');
+
+// console.log('MONGODB_URI:', process.env.MONGODB_URI);
+
+
+const uri = process.env.MONGODB_URI;
```

Comment:
Please see other files on how to do this, we do not use `process`

Reply:
Done!